wire-ios/Wire-iOS/Sources/Developer/DeveloperTools/DeepLinks/DeepLinksViewModel.swift

```diff
@@ -67,4 +79,27 @@ final class DeepLinksViewModel: ObservableObject {
             _ = self.router?.openDeepLinkURL(url)
         }
     }
+
+    func openSwitchBackendLink(for backend: Backend) {
+        let config: String
+        switch backend {
+        case .staging:
+            config = "https://staging-nginz-https.zinfra.io/deeplink.json"
+        case .anta:
+            config = "https://nginz-https.anta.wire.link/deeplink.json"
+        case .bella:
+            config = "https://nginz-https.bella.wire.link/deeplink.json"
+        case .chala:
+            config = "https://nginz-https.chala.wire.link/deeplink.json"
+        case .diya:
+            config = "https://nginz-https.diya.wire.link/deeplink.json"
+        case .elna:
+            config = "https://nginz-https.elna.wire.link/deeplink.json"
+        case .foma:
+            config = "https://nginz-https.foma.wire.link/deeplink.json"
+        }
```

Comment:
```suggestion
        case .lich:
            config = "https://nginz-https.lich.wire.link/deeplink.json"
        }
```

or should we do: 
```
default:
  config = "https://nginz-https.\(config.rawValue).wire.link/deeplink.json"
  ```
  for most of the cases

Reply:
done